operations/agent-flow-mixin/alerts/clustering.libsonnet

```diff
@@ -84,10 +61,7 @@ alert.newGroup(
         ) > 1
       |||,
       'Cluster nodes are not using the same configuration file.',
-      '5m',
+      '10m',
```

Comment:
I believe it doesn't hurt to have the 5m interval; deploying an erroneous configuration file or being stuck in terminating state doesn't sound something that can be flappy.

Reply:
Happy to revert for this one alert I guess, the config map reloader not kicking in for 5min could be a permanent issue, yeah.